docs/pages/guides/nextjs-starter-kit.mdx

```diff
@@ -0,0 +1,545 @@
+server side and client side methods which contain the logic to create and modify
+documents.
+
+<br />
+<details>
+  <summary>More info about the structure</summary>
+  <br />
+  <dl>
+    <dt>`/components/`</dt>
+    <dd>Components used in the app.</dd>
+    <dt>`/data/`</dt>
+    <dd>A demo database.</dd>
+    <dt>`/layouts/`</dt>
+    <dd>Page layout components.</dd>
+    <dt>`/lib/client/`</dt>
+    <dd>Client functions used for accessing documents, database, SWR hooks.</dd>
+    <dt>`/lib/server/`</dt>
+    <dd>
+      Server functions used for accessing documents, database, liveblocks.
+    </dd>
+    <dt>`/pages/`</dt>
+    <dd>Next.js pages.</dd>
+    <dt>`/primitives/`</dt>
+    <dd>Generic reusable components.</dd>
+    <dt>`/types/`</dt>
+    <dd>All reusable TypeScript types.</dd>
+    <dt>`/liveblocks.config.ts`</dt>
+    <dd>Liveblocks config file.</dd>
+    <dt>`/liveblocks.server.config.ts`</dt>
+    <dd>Server-side Liveblocks config file.</dd>
+  </dl>
+</details>
+
+### Liveblocks Configuration
+
+If you already have a Liveblocks Development account, you may skip to step 4.
+
+1. Create an account at [liveblocks.io](https://liveblocks.io])
+2. Navigate to the [dashboard](https://liveblocks.io/dashboard) and click "Add
+   project..."
+3. Create a new project with "Development" as the environment.
+4. Go to "API keys" on the side menu, reveal your secret key, and copy it
+
+<Figure>
+  <video autoPlay loop muted playsInline>
+    <source
+      src="/images/docs/next-js-starter-kit/create-api-key.mp4"
+      type="video/mp4"
+    />
+  </video>
+</Figure>
+
+5. Install the starter kit by running
+
+```shell
+npx create-liveblocks-app --next
+```
+
+6. Enter the `.env.local` file, which is located alongside `package.json` at the
+   root level in your project.
+7. Place your secret key into the file as `LIVEBLOCKS_SECRET_KEY`.
+
+Your `.env.local` file should contain a line similar to this:
+
+```dotenv file=".env.local"
+LIVEBLOCKS_SECRET_KEY =
+  sk_dev_XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX;
+```
+
+At this point, Liveblocks can be run with
+
+```shell
+npm run dev
+```
+
+### Authentication
+
+The Liveblocks starter kit uses [NextAuth.js](https://next-auth.js.org/) for
+authentication, meaning many authentication providers can be configured with
+minimal code changes. A demo authentication system is used by default, but it's
+easy to add real providers, such as GitHub, Auth0, and more.
+
+Take a look at the guide for your chosen authentication method:
+
+- [GitHub authentication](#github-authentication)
+- [Auth0 authentication](#auth0-authentication)
+- [Demo authentication (skip to next section)](#how-to-sign-up)
+
+### GitHub authentication [#github-authentication]
+
+To use GitHub auth, make sure you selected "GitHub authentication" when running
+the installer (or you've set up the
+[provider manually](#add-multiple-authentication-providers)). This is how to set
```

Comment:
"This is how to set up your secret keys."

In a previous section, we mention a very similar thing: "Place your secret key into the file as `LIVEBLOCKS_SECRET_KEY`."
It might be confusing and for impatient people, they might think they've already done it before

Could we clarify here? “This is how to set up your **GitHub** secret keys."
Same for Auth0